site/docs/latest/security/authorization.md

```diff
@@ -74,36 +74,19 @@ needs to be able to publish to other clusters' topics.
 
 ## Pulsar admin authentication
 
-```java
-String authPluginClassName = "com.org.MyAuthPluginClass";
-String authParams = "param1:value1";
-boolean useTls = false;
-boolean tlsAllowInsecureConnection = false;
-String tlsTrustCertsFilePath = null;
-
-ClientConfiguration config = new ClientConfiguration();
-config.setAuthentication(authPluginClassName, authParams);
-config.setUseTls(useTls);
-config.setTlsAllowInsecureConnection(tlsAllowInsecureConnection);
-config.setTlsTrustCertsFilePath(tlsTrustCertsFilePath);
-
-PulsarAdmin admin = new PulsarAdmin(url, config);
-```
-
 To use TLS:
 
 ```java
+String url = "https://my-broker.example.com:8443";
 String authPluginClassName = "com.org.MyAuthPluginClass";
```

Comment:
This isn't the TLS plugin name. Should be org.apache.pulsar.client.impl.auth.AuthenticationTls.
Similarly, params should be tlsCertFile:/path/to/client-cert.pem,tlsKeyFile:/path/to/client-key.pem